internal/command.go

```diff
@@ -7,10 +7,9 @@ import (
 	"strings"
 
 	shell "github.com/brianstrauch/cobra-shell"
-	"github.com/spf13/cobra"
-
 	ccloudv1 "github.com/confluentinc/ccloud-sdk-go-v1-public"
 	cliv1 "github.com/confluentinc/ccloud-sdk-go-v2/cli/v1"
+	"github.com/spf13/cobra"
```

Comment:
The cobra import needs to go under line 9 because it's a non-Confluent dependency